server/featureflags/server/web/backend.py

```diff
@@ -58,29 +76,87 @@ async def index(_):
                                  'static/index.html').decode('utf-8'))
 
 
-async def call(request):
-    request_proto = Request.FromString(request.body)
-
-    if request_proto.operations:
+def graph_context(
+    sa_engine,
+    session,
+    ldap,
+    ctx=None
+):
+    ctx_ = {
+        SA_ENGINE: sa_engine,
+        SESSION: session,
+        LDAP: ldap,
+        DIRTY: DirtyProjects(),
+        CHANGES: Changes(),
+        IDS: dict(),
+    }
+
+    if ctx is not None:
```

Comment:
`ctx` and `ctx_` naming is a little bit frustrating - they look pretty same and it will be very easy to misuse of of them  
I'd recommend to change this to something `existing_ctx` and `new_ctx` or something like that

Reply:
replaced with ctx_override